ibm/resource_ibm_container_vpc_cluster.go

```diff
@@ -973,67 +963,38 @@ func resourceIBMContainerVpcClusterDelete(d *schema.ResourceData, meta interface
 	}
 
 	if region != "" {
-		userDetails, err := meta.(ClientSession).BluemixUserDetails()
+
 		if err != nil {
```

Comment:
Can you remove this if err since we removed 
userDetails, err := meta.(ClientSession).BluemixUserDetails()

Reply:
Yes, made the required changes and pushed the code